src/EndpointsControllers/defaultUserController.js

```diff
@@ -0,0 +1,74 @@
+import sql from 'mysql-bricks';
+import { handleQuery } from '../Helpers/index.js';
+
+export class DefaultUserController {
+    constructor (tableName) {
+        this._tableName = tableName;
```

Comment:
Try making this controller independent, meaning that it will not need a table name later. If it's user controller - the table it works with should be `users`